pom.xml

```diff
@@ -721,6 +721,9 @@
             <artifactId>java16</artifactId>
             <version>1.1</version>
           </signature>
+          <annotations>
+            <annotation>io.netty.util.SuppressForbidden</annotation>
```

Comment:
Idea is to more finely target suppression of the animal sniffer errors, but I agree that reflection is better; undoing and fixing.